### ISSUE TITLE
Unknown file found

### ISSUE THREAD

**Issue by guitaristtom** [Inrixia/Floatplane-Downloader#108]
I keep getting this spammed in the log for the container:

```
Checking for files older than 14 days in /fp/videos for deletion...

!!WARNING!! - Found file for deletion in /fp/videos that is not a mp4, nfo or png file!
This could indicate that the root video folder is not being detected properly.
Please report this as a Issue at https://github.com/Inrixia/Floatplane-Downloader/issues/new
```

After doing some digging I managed to find this in my `Linus Tech Tips` folder:
`.fuse_hidden0003e98f0000037e`

There was another one with a similar name in TechLinked.

One improvement I might suggest is to output the list of files that is causing this issue, it would make debugging this kind of thing easier for the future.

**Comment by Inrixia** [comment]
This is a good suggestion and something I will add. This is in there because deletion is done by the file modified date and looks for nfo, png and mp4's. So if you point it at a directory that contains other things it may indicate its seeing files it shouldn't.

Not a great implementation but the best way to handle it for now.

I'll definitely add a file printout for ones its seeing though. Thanks

**Comment by guitaristtom** [comment]
Holy cow, I wasn't expecting a reply so quickly. You're on the ball. Thank you.

I just deleted both of those `.fuse_hidden*` files and the container starts up fine now. I guess this issue was more to just bring it to your attention and to give that suggestion.

_If it helps_ at all for debugging, I used this command when I changed to the /fp/videos directory:
`ls -lah -I "*.mp4" -I "*.nfo" -I "*.png" *`
